terraform/modules/jenkins/iam.tf

```diff
@@ -50,6 +50,7 @@ data "aws_iam_policy_document" "jenkins_fargate_policy_document" {
       "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/TDRJenkinsNodeLambdaRoleProd",
       "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/TDRJenkinsNodeLambdaRoleIntg",
       "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/TDRJenkinsPublishRole",
+      "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/TDRTerraformRoleMgmt",
       "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/TDRJenkinsNodeReadParamsRoleIntg",
```

Comment:
This isn't much to do with the PR, it was inserted manually and I suspect we need it.